man/man4/zfs.4

```diff
@@ -595,7 +595,7 @@ Under Linux, half of system memory will be used as the limit.
 Under
 .Fx ,
 the larger of
-.Sy all_system_memory - 1GB No and Sy 5/8 * all_system_memory
+.Sy all_system_memory No - Sy 1GB No and Sy 5/8 No \(md Sy all_system_memory
```

Comment:
` No \(md Sy` does not render as an `*` for me with man v2.7.6.1, but instead as an empty square.  This was with RHEL 8.

Reply:
`md` is the multiplication dot – are you on a VT?